hack/common.sh

```diff
@@ -50,8 +50,8 @@ OCS_CSV="$OUTDIR_TEMPLATES/ocs-operator.csv.yaml.in"
 # and we should continue to use dowsntream image only due to few
 # downstream only changes present in rook downstream fork.
 LATEST_ROOK_IMAGE="quay.io/ocs-dev/rook-ceph:vrelease-4.17-91cc5780b"
-LATEST_NOOBAA_CORE_IMAGE="quay.io/noobaa/noobaa-core:master-20240820"
-LATEST_NOOBAA_DB_IMAGE="docker.io/centos/postgresql-12-centos8"
+LATEST_NOOBAA_CORE_IMAGE="quay.io/noobaa/noobaa-core:master-20240901"
+LATEST_NOOBAA_DB_IMAGE="quay.io/sclorg/postgresql-15-c9s"
```

Comment:
Just adding this link here for future reference
https://github.com/noobaa/noobaa-operator/blob/master/pkg/options/options.go#L73-L75